### PR TITLE
DocSite Improvements: Acknowledgement section and better images

### PR DESCRIPTION
**Pull request by Tanmay-97**
Reviewer: @suzinyou  

Estimate: 5m

### Changes
- Uses an IDi favicon
- Increased topleft logo size
- Switched to "Inter" font
- Standardized logo image and enhanced colors
- Added acknowledgement section
- Added privacy policy
- changed link hover colors

### CODE REVIEW

File: docs/stylesheets/extra.css [hunk 1]
```diff
@@ -9,27 +13,28 @@
...
 .link-home {
   color: #c99300 !important;
 }
-
+.link-home:hover {
+  color: #5480cb !important;
+}
 .md-typeset a {
   color: #5480cb;
 }
```
Comment (suzinyou) [inline docs/stylesheets/extra.css:34]: I know this isn't part of the PR but noticed that the strong-faced font & icons look quite light in Light mode! 
image

Also, this blue color feels like it should be a link. I think that's what visitors might be used to also?

File: docs/images/idinsight_org.png
Comment (suzinyou) [inline docs/images/idinsight_org.png:1]: It looks great in dark mode, but wonder if we can switch to the navy one in light mode? 
image

File: docs/index.md [hunk 2]
```diff
@@ -51,7 +51,14 @@
...
 The project is funded by Google.org through the
 <a href="https://globalgoals.withgoogle.com/globalgoals/" class="link-home">AI for Global Goals</a> grant.
+<br><br>
+<img src="./images/idinsight_org.png" alt="idi_org" width=200 style="margin-left:-25px" /><br>
+Built and powered by IDinsight.
+<p class="footer" markdown>IDinsight uses data and evidence to help leaders combat poverty worldwide. Our collaborations deploy a large analytical toolkit to help clients design better policies, rigorously test what works, and use evidence to implement effectively at scale. We place special emphasis on using the right tool for the right question, and tailor our rigorous methods to the real-world constraints of decision-makers. IDinsight works with governments, foundations, NGOs, multilaterals and businesses across Africa and Asia. We work in all major sectors including health, education, agriculture, governance, digital ID, financial access, and sanitation. We have offices in Dakar, Lusaka, Manila, Nairobi, New Delhi, Rabat, and Remote.
+<br>
+<br>
+:globe_with_meridians: <a href="https://www.idinsight.org" class="link-home">www.idinsight.org</a>
```
Comment (suzinyou) [inline docs/index.md:63]: This is also maybe out of scope for this PR and don't need to address this point, but this light-dark mode colors seem tricky. The yellow looks great in dark mode but a bit low contrast for light mode 🤔 
image

**Review by suzinyou — APPROVED** [review]
Thank you! Looks great in dark mode, but a few things seemed too light for the light mode -- I know this might depend on Lenny's input so feel free to just merge as is

File: docs/stylesheets/extra.css (hunk 1 above)
Comment (Tanmay-97) [reply docs/stylesheets/extra.css:34]: I agree, ticketing this. I was unable to figure a fix

File: docs/images/idinsight_org.png
Comment (Tanmay-97) [reply docs/images/idinsight_org.png:1]: Fixed

**Pull request merged**